ocs_ci/ocs/ui/block_pool.py

```diff
@@ -364,3 +366,102 @@ def check_storage_class_attached(self, pool_name):
             f"Pool name {pool_name} has {storage_class_attached} storageclass attached to it."
         )
         return int(storage_class_attached)
+
+    def pool_raw_capacity_loaded(self, pool_name):
+        """
+        Takes pool name and returns True if the raw capacity of the block pool is loaded
+        or returns False if the capacity is not loaded.
+
+        Args:
+            pool_name (str): The name of the pool to be deleted
+
+        Returns:
+            bool: True if raw capacity of the blockpool is loaded, otherwise False
+
+        """
+        logger.info("Checking if the Block Pool Raw Capacity has loaded in UI")
+
+        self.select_blockpool(pool_name)
+
+        bf_obj = BlockAndFile()
+        _, raw_capacity_loaded = bf_obj.get_raw_capacity_card_values()
+
+        if raw_capacity_loaded:
+            logger.info("Block Pool Raw Capacity has loaded in UI")
+        else:
+            logger.warning("Block Pool Raw Capacity has not loaded in UI")
+        return raw_capacity_loaded
+
+    def cross_check_raw_capacity(self, pool_name):
+        """
+        Takes pool name and returns True if the raw capacity of the block pool is same in GUI as obtained from CLI
+        or returns False if the raw capacity of the block pool doesnt match with CLI
+
+        Args:
+            pool_name (str): The name of the pool to be deleted
+
+        Returns:
+            bool: True if raw capacity of the blockpool is is same in GUI as obtained from CLI, otherwise False
+
+        """
+        logger.info(
+            "Checking if the Block Pool Raw Capacity is same in GUI as obtained from CLI"
+        )
+        if self.pool_raw_capacity_loaded(pool_name):
+            cmd = f"rados df --pool={pool_name}"
+            ct_pod = pod.get_ceph_tools_pod()
+            df_op = ct_pod.exec_cmd_on_pod(command=cmd)
+            logger.info(f"{df_op=}")
+            # splitting the ouptut with spaces
+            # the blockpool used capacity will always be at index 17 and 18
+            # where the index 17 is the value and index 18 is the unit
+            used_capacity_in_CLI, unit = df_op.split()[17:19]
```

Comment:
[17:19] - is this aways the case, even when numbers and units are different? I think we need some intellectual parsing here to get numbers from the text. Please use regex-like to get what you need from the string

Reply:
Hello Daniel, I have checked it for numerous cases and its always the same, thus I have added the explanation in the comments, and the split is made with spaces, so I do not think any of the units will be affected as they do not have any space in between their unit names. 
Regex would be a better option if we would always know the pool name or the unit names so that we can group it using look forward or look behind functions. And here those two would always be different. And even if we create a regex with all units it would be very complicated regex expression to be understood by others, thus after a lot of thoughts I have added a simple split and indexing method that is easy to understand and use.

Reply:
ack